### PR TITLE
Don't install BouncyCastleProvider globally, instead use it explicitly where needed

### PR DESCRIPTION
**Pull request by LukasGasior1**

### CODE REVIEW

File: core/src/main/java/com/radixdlt/RadixNodeApplication.java [hunk 1]
```diff
@@ -86,32 +84,13 @@ private RadixNodeApplication() {}
...
     try {
       MemoryLeakDetector.start();
-
       logVersion();
       dumpExecutionLocation();
-      // Bouncy Castle is required for loading the node key, so set it up now.
-      setupBouncyCastle();
       final var properties = RuntimeProperties.fromCommandLineArgs(args);
       bootstrapRadixNode(properties);
-      log.info("Node bootstrapping completed");
```
Comment (dhedey) [inline core/src/main/java/com/radixdlt/RadixNodeApplication.java:114]: Out of interest, why was this removed?

File: core/src/main/java/com/radixdlt/RunningRadixNode.java [hunk 2]
```diff
@@ -96,8 +96,6 @@ private RunningRadixNode(Injector injector) {
   }
 
   public static RunningRadixNode run(UnstartedRadixNode unstartedRadixNode) {
-    log.info("Starting Radix node");
```
Comment (dhedey) [inline core/src/main/java/com/radixdlt/RunningRadixNode.java:99]: Out of interest, why was this removed?

**Review by dhedey — APPROVED** [review]
Approve of the BouncyCastle changes, and state manager log changes.

HostIp changes look okay - assuming they've been sufficiently tested

File: core/src/main/java/com/radixdlt/RunningRadixNode.java (hunk 2 above)
Comment (LukasGasior1) [reply core/src/main/java/com/radixdlt/RunningRadixNode.java:99]: too many open branches :sweat_smile: I've replaced this with `https://github.com/radixdlt/babylon-node/pull/548/files#diff-d14e0db8dc21558566c417273e02c97a19d29b5502fd22260c256bdbb412dbb6R178` in a different PR

File: core/src/main/java/com/radixdlt/RadixNodeApplication.java (hunk 1 above)
Comment (LukasGasior1) [reply core/src/main/java/com/radixdlt/RadixNodeApplication.java:114]: we already have a similar message in RadixNodeApplication:
```Radix node {} started successfully in {} ms```
